src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/RunCodeActionService.cs

```diff
@@ -43,16 +44,19 @@ public RunCodeActionService(OmnisharpWorkspace workspace, [ImportMany] IEnumerab
             var operations = await action.GetOperationsAsync(CancellationToken.None);
 
             var solution = _workspace.CurrentSolution;
+            IEnumerable<OmniSharp.Models.ModifiedFileResponse> changes = Enumerable.Empty<OmniSharp.Models.ModifiedFileResponse>();
+            var directoryName = Path.GetDirectoryName(request.FileName);
             foreach (var o in operations)
             {
-                o.Apply(_workspace, CancellationToken.None);
+                var applyChangesOperation = o as ApplyChangesOperation;
```

Comment:
To answer the original question, now that I've looked there a bunch of refactorings that are disabled in `CodeActionHelper`.   It appears they are filtered out from the initial list.